### PR TITLE
Some README improvements

### LINKED ISSUE #3: Create Quay repository

### PR DESCRIPTION
**Pull request by mzylowski**
- Information about kubernetes-entrypoint docker image
- Improved daemonset example

Closes #3 
Related with #15

cc @DTadrzak @PiotrProkop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,8 +46,10 @@ Example:
 ### Daemonset
 Checks if a specified daemonset is already running on the same host, this dependency requires a `POD_NAME`
 env which can be easily passed through the [downward api](http://kubernetes.io/docs/user-guide/downward-api/).
+The `POD_NAME` variable is used to determine right host to check dependencies.
```
Comment (PiotrProkop) [inline README.md]: I would change it to :
The `POD_NAME` variable is **mandatory** and is used to resolve dependencies.

File: README.md [hunk 2]
```diff
@@ -46,8 +46,10 @@ Example:
...
 Example:
 
+`POD_NAME=nova`
```
Comment (PiotrProkop) [inline README.md]: I would delete this POD_NAME has to passed via downward api.

File: README.md [hunk 3]
```diff
@@ -73,7 +75,12 @@ Example:
 
 `DEPENDENCY_SOCKET=/var/run/openvswitch/ovs.socket`
 
+## Image
+
+Image is build after every push to kubernetes-entrypoint repository.
```
Comment (PiotrProkop) [inline README.md]: I would change it to :
Build process for image is trigged after each commit.

File: README.md [hunk 4]
```diff
@@ -46,6 +46,7 @@ Example:
 ### Daemonset
 Checks if a specified daemonset is already running on the same host, this dependency requires a `POD_NAME`
 env which can be easily passed through the [downward api](http://kubernetes.io/docs/user-guide/downward-api/).
+The `POD_NAME` variable is mandatory and is used to resolve dependencies.
 Example:
```
Comment (PiotrProkop) [inline README.md]: Add example  of using downward api.

**Comment by mzylowski** [comment]
Added link to example how to use downward api 
[example](https://raw.githubusercontent.com/kubernetes/kubernetes.github.io/master/docs/user-guide/downward-api/dapi-pod.yaml) 
@PiotrProkop PTAL

**Comment by PiotrProkop** [comment]
LGTM

**Pull request merged**